addons/spreadsheet/static/tests/lists/list_plugin_test.js

```diff
@@ -252,8 +252,6 @@ QUnit.module("spreadsheet > list plugin", {}, () => {
     QUnit.test("user context is combined with list context to fetch data", async function (assert) {
         const context = {
             allowed_company_ids: [15],
-            default_stage_id: 5,
```

Comment:
Why those changes ?

Reply:
because this context is used as the `user_context` of the current `session`, and `default_stage_id` and `search_default_stage_id` are not part of the `userContext`

The purpose of the test is to check whether the `userContext` keys are removed from de spreadSheetData,

and that the resulting context is the default `context` overrided with the `context from spreadSheetData` minus the keys from the `userContext`

So even with my change, the test still checks what it is supposed to.

(if we wanted to we could think of a way to add `default_stage_id` and `search_default_stage_id` to the global context, but the test would have to be modified further)